obspy/io/nordic/core.py

```diff
@@ -0,0 +1,1379 @@
+                                                int(topline[19:20]) *
+                                                100000)\
+            + add_seconds
+    except:
+        f.close()
+        IOError("Couldn't read a date from sfile")
+    # new_event.loc_mod_ind=topline[20]
+    new_event.event_descriptions.append(EventDescription())
+    new_event.event_descriptions[0].text = topline[21:23]
+    # new_event.ev_id=topline[22]
+    if not _float_conv(topline[23:30]) == 999:
+        new_event.origins[0].latitude = _float_conv(topline[23:30])
+        new_event.origins[0].longitude = _float_conv(topline[31:38])
+        new_event.origins[0].depth = _float_conv(topline[39:43]) * 1000
+    else:
+        # The origin 'requires' a lat & long
+        new_event.origins[0].latitude = float('NaN')
+        new_event.origins[0].longitude = float('NaN')
+        new_event.origins[0].depth = float('NaN')
+    # new_event.depth_ind = topline[44]
+    # new_event.loc_ind = topline[45]
+    new_event.creation_info = CreationInfo(agency_id=topline[45:48].
+                                           strip())
+    ksta = Comment(text='Number of stations=' +
+                   topline[49:51].strip())
+    new_event.origins[0].comments.append(ksta)
+    # new_event.origins[0].nsta??? = _int_conv(topline[49:51])
+    if not _float_conv(topline[51:55]) == 999:
+        new_event.origins[0].time_errors['Time_Residual_RMS'] = \
+            _float_conv(topline[51:55])
+    # Read in magnitudes if they are there.
+    if len(topline[59].strip()) > 0:
+        new_event.magnitudes.append(Magnitude())
+        new_event.magnitudes[0].mag = _float_conv(topline[56:59])
+        new_event.magnitudes[0].magnitude_type = topline[59]
+        new_event.magnitudes[0].creation_info = \
+            CreationInfo(agency_id=topline[60:63].strip())
+        new_event.magnitudes[0].origin_id = new_event.origins[0].\
+            resource_id
+    if len(topline[67].strip()) > 0:
+        new_event.magnitudes.append(Magnitude())
+        new_event.magnitudes[1].mag = _float_conv(topline[64:67])
+        new_event.magnitudes[1].magnitude_type = topline[67]
+        new_event.magnitudes[1].creation_info = \
+            CreationInfo(agency_id=topline[68:71].strip())
+        new_event.magnitudes[1].origin_id = new_event.origins[0].\
+            resource_id
+    if len(topline[75].strip()) > 0:
+        new_event.magnitudes.append(Magnitude())
+        new_event.magnitudes[2].mag = _float_conv(topline[72:75])
+        new_event.magnitudes[2].magnitude_type = topline[75]
+        new_event.magnitudes[2].creation_info = \
+            CreationInfo(agency_id=topline[76:79].strip())
+        new_event.magnitudes[2].origin_id = new_event.origins[0].\
+            resource_id
+    # convert the nordic notation of magnitude to more general notation
+    for _magnitude in new_event.magnitudes:
+        _magnitude.magnitude_type = _nortoevmag(_magnitude.magnitude_type)
```

Comment:
I don't know why this couldn't be done on the line in which it was assigned above.